acestep/ui/gradio/events/generation/metadata_loading.py

```
@@ -143,15 +157,17 @@ def load_metadata(file_obj, llm_handler=None):
             track_name, complete_track_classes, instrumental,
             retake_variance, retake_seed,
+            edit_target_caption, edit_target_lyrics,
+            edit_n_min, edit_n_max, edit_n_avg,
             True  # is_format_caption
         )
 
     except json.JSONDecodeError as e:
         gr.Warning(t("messages.invalid_json", error=str(e)))
-        return [None] * 42 + [False]
+        return [None] * 47 + [False]
     except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)